roles-in-tech/developer-advocate.md

```diff
@@ -2,89 +2,146 @@
+- It bridges age and culture gaps — if you’re not interested in a cutthroat competition in engineering, you have a chance to use your talent otherwise.
+Before you jump on the opportunity, there are some very important points to remember:
+- Developer relations is not a starting position — most developer evangelists graduated from being developers in the same company. You need to know the pain to help prevent it.
+- There are part time opportunities though — engineers or people learning in the company can help with Devrel to ease into the job earlier.
+- Always be ready to prove your worth — measuring the impact of a developer evangelist is tough, you need to make sure you’re well organised in recording your successes.
+It’s a versatile, morphing and evolving role. 
 
-### Objectives
 
-**Participants will be able to:**
+### Objectives
 
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
+Developer Advocate is a software-focused technical person who has good communication and community-building skills.  The responsibilities of the role typically are some or all of the following:
+1) Acting as a steward of open-source projects that the company starts or sponsors, including performing code reviews, contributing code to the project, and helping drive the direction of that project.
+2) Writing public, open-source libraries to wrap APIs for the company's products and helping to integrate those into existing open source projects that might make use of them (for instance the work from Rackspace on Apache libcloud which helped to get Rackspace Cloud integration into SaltCloud and other tools).
+3) Being a consultative source of information for managers within the company to help them understand the importance of the projects to which you are assigned and the role that open-source plays within it.
+4) A participant in product-focused development planning and sprint meetings as an advocate for developer customers to ensure that issues in the product which result in bugs being opened in open-source software that uses your product or are brought to your attention within the developer community are addressed appropriately.
+5) Integration into a product team to assist them with understanding the developer customer use case for the product and driving that direction in API development, documentation, feature development, and test processes.
 
 ### Specific Things To Teach
 
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+As a translator, you have both outbound (company to people) and inbound (people to your company) tasks to fulfil. Let’s list most of them:
+Outbound tasks
+- Be a social media presence
+- Keep up to date with competition and market
+- Create openly available software products
+- Participate in other products
```

Comment:
This should be a bulleted list of things you will show them in the lesson and practices. The rest should go under “lesson”. You are welcome to make sub-headers in “lesson”.